source/mainnet/net/guides/developer-page.rst

```diff
@@ -7,22 +7,11 @@ Developer resources
 
 The following developer resources help you get started developing on the Concordium network.
 
-Official resources
-==================
+All of our repositories are on `GitHub <https://github.com/Concordium>`__.
 
-- See :ref:`Node setup<node-requirements>` for information about the requirements to run a node and the available platforms for nodes.
+.. dropdown:: Standards
 
-- See :ref:`SDKs and APIs<sdks-apis>` for links to SDKs and APIs for development
-
-- See :ref:`Smart contracts <introduction>` for general information about smart contracts
-
-    - `Smart contract libraries <https://crates.io/crates/concordium-std>`_
-
-    - To ease deployment and initialization, you can use the `Smart contract deploy and initialize tool <https://sctools.mainnet.concordium.software/>`__ instead of the process below. It works with the |bw| to deploy and initialize smart contracts to Mainnet and Testnet.
-
-- `GitHub <https://github.com/Concordium>`_
-
-- `Concordium standards and updates <https://proposals.concordium.software/>`_
+    - `Concordium standards and updates <https://proposals.concordium.software/>`_
 
     - `CIS-0 standard detections <https://proposals.concordium.software/CIS/cis-0.html>`_
 
```

Comment:
**Suggestion:** I don't think we should have a  '- Applicable for smart contracts v1', every standard except CIS-1 is, and CIS-1 is deprecated. So we should probably just list the standards in the top list and avoid this sub list.